etna/series/models.py

```diff
@@ -0,0 +1,46 @@
+from django.db import models
+
+from wagtail.admin.panels import FieldPanel
+from wagtail.search import index
+
+from wagtailmetadata.utils import get_image_model_string
+
+
+class SeriesCategory(models.TextChoices):
+    RECORD_COLLECTION = "Record collection", "Record collection"
+
+
+class Series(index.Indexed, models.Model):
+    admin_name = models.CharField(
+        max_length=128,
+        help_text="The name of the series as it should be displayed in the CMS site.",
+    )
+    category = models.CharField(
+        choices=SeriesCategory.choices,
+        blank=True,
+    )
+    image = models.ForeignKey(
+        get_image_model_string(), on_delete=models.SET_NULL, null=True, blank=False
+    )
+
+    # TODO: Once the real integration with the CIIM search is built,
+    #       we should add a field that will allow identify the series
+    #       in the CIIM search and on record detail pages.
```

Comment:
**Thoughts:** I'm not sure if series should be a taxonomy model at all. It does feel like it as you will want to tag content with the taxonomy. Also, in the future, this will be required to build the pre-filtered search links and link page content back on the record detail pages.

Reply:
All sounds pretty good to me! I don't think we will need the `category` field though, but may have missed a discussion around this.